test/int/sources/sources.noBreakpoints.test.ts

```diff
@@ -0,0 +1,71 @@
+import * as path from 'path';
+import * as testSetup from '../testSetup';
+import * as assert from 'assert';
+import { puppeteerSuite, puppeteerTest } from '../puppeteer/puppeteerSuite';
+import { TestProjectSpec } from '../framework/frameworkTestSupport';
+import * as sourcesHelper from './sources.utils';
+import { LoadedSourceEvent } from 'vscode-debugadapter';
+
+// We are using the "Simple" project app inside the testdata directory
+const DATA_ROOT = testSetup.DATA_ROOT;
+const SIMPLE_PROJECT_ROOT = path.join(DATA_ROOT, 'simple');
+const TEST_SPEC = new TestProjectSpec( { projectRoot: SIMPLE_PROJECT_ROOT } );
+const sourceChecker = new sourcesHelper.SourcesChecker();
+
+puppeteerSuite('Static HTML Pages Without Breakpoints', TEST_SPEC, (suiteContext) => {
+    puppeteerTest('Receive loaded source events after page load', suiteContext, async (context, page) => {
+        sourceChecker.resetLoadedSourcesCount();
```

Comment:
I think that creating a new instance for each tests is less error-prone than having to reset it.